recipe/meta.yaml

```diff
@@ -42,13 +44,18 @@ test:
     - gi-docgen --help
 
 about:
-  home: The package home page
-  license: "Apache Software License or GNU General Public v3 or later (GPLv3+)"
-  license_family: GPL3
-  license_file:
-  summary: "Documentation tool for GObject-based libraries"
-  doc_url:
-  dev_url:
+  home: https://gitlab.gnome.org/GNOME/gi-docgen
+  summary: Documentation tool for GObject-based libraries
+  description: |
+    GI-DocGen is a document generator for GObject-based libraries. GObject is
+    the base type system of the GNOME project. GI-Docgen reuses the
+    introspection data generated by GObject-based libraries to generate the API
+    reference of these libraries, as well as other ancillary documentation.
+  license: GPL-3.0-or-later
```

Comment:
Licenses are: `GPL-3.0-or-later AND Apache-2.0 AND CC0-1.0`, see https://gitlab.gnome.org/GNOME/gi-docgen/-/blob/2022.1/pyproject.toml#L13 or https://gitlab.gnome.org/GNOME/gi-docgen/-/blob/2022.1/setup.cfg#L20-21 or https://gitlab.gnome.org/GNOME/gi-docgen/-/blob/2022.1/setup.py#L69